src/helpers/result.ts

```diff
@@ -63,24 +69,28 @@ export function formatMultiResult(result: DecodedMultiResult): string {
   let formatted = `${result.solved}/${result.attempted}`;
   if (result.centiseconds && result.centiseconds !== 9999900) {
     let fcs = formatCentiseconds(result.centiseconds);
-    formatted += ` ${ fcs.substring(0, fcs.length - 3) }`; // chop off the decimal part
+    formatted += ` ${fcs.substring(0, fcs.length - 3)}`; // chop off the decimal part
   }
   return formatted;
 }
 
-function isDnfMultiResult(result: DecodedMultiResult): result is DnfMultiResult {
+function isDnfMultiResult(
+  result: DecodedMultiResult
+): result is DnfMultiResult {
   return result.hasOwnProperty('isDnf');
 }
 
-function isDnsMultiResult(result: DecodedMultiResult): result is DnsMultiResult {
+function isDnsMultiResult(
+  result: DecodedMultiResult
+): result is DnsMultiResult {
   return result.hasOwnProperty('isDns');
 }
 
 function decodeOldMultiResult(result: AttemptResult): DecodedMultiResult {
-  let r = `${result}`;
-  let ss = parseInt(r.substr(1, 2), 10);
-  let aa = parseInt(r.substr(3, 2), 10);
-  let tt = parseInt(r.substr(5, 5), 10);
+  let r = result.toString();
+  let ss = parseInt(r.substring(1, 3), 10);
+  let aa = parseInt(r.substring(3, 5), 10);
+  let tt = parseInt(r.substring(5, 10), 10);
```

Comment:
Instead of `substring` you can use division reminder, I think it ends up cleaner. wca-live has [an implementation](https://github.com/thewca/wca-live/blob/16b362ea0caf38c6843632c02363d59ee052e781/client/src/lib/attempt-result.js#L125-L140), if you want to have a look :)